.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,86 @@
+# Workflow: Generate SBOM
+# This workflow generates a Software Bill of Materials (SBOM) using the CycloneDX Maven plugin.
+# The SBOM provides a list of dependencies and helps track potential security risks.
+# The workflow runs on a schedule for testing purposes and can be manually triggered.
+# The generated SBOM is uploaded as an artifact for further analysis or distribution.
+
+name: Generate SBOM
+
+on:
+  workflow_dispatch:
+  pull_request: # Triggers the workflow on pull requests
```

Comment:
Why are we triggering in the pull request again? We had removed it in the final PR https://github.com/camunda/camunda/pull/23070/files

Reply:
Ah, I guess you added it again to test?

Reply:
Yes :) Will remove it before the review